src/scss/components/_product-detail.scss

```diff
@@ -0,0 +1,188 @@
+.product-detail {
+  background-color: var(--gray);
+  padding-top: 0;
+  padding-bottom: 112px;
+
+  @include breakpoint(sm) {
+    padding-top: 52px;
+  }
+
+  .col-sm-6,
+  .col-sm-4 {
+    @include breakpoint(sm) {
+      padding: 0;
+    }
+  }
+
+  &__images {
+    display: flex;
+    width: 92vw;
+    margin: 4vw;
+    overflow-x: auto;
+    -webkit-overflow-scrolling: touch;
+    scroll-snap-type: x mandatory;
+    scroll-behavior: smooth;
+    scrollbar-width: none;
+
+    @include breakpoint(sm) {
+      width: 100%;
+      margin: 0;
+      overflow-x: visible;
+      flex-flow: row wrap;
+      align-items: flex-start;
+      align-content: flex-start;
+      margin-bottom: 40px;
+      gap: 1rem;
+    }
+
+    @include breakpoint(lg) {
+      gap: 2rem;
+    }
+  }
+
+  &__images::-webkit-scrollbar {
+    display: none;
+    scrollbar-width: none;
+  }
+
+  &__images > div {
+    scroll-snap-align: start;
+  }
+
+  &__image {
+    display: flex;
+    flex-shrink: 0;
+    justify-content: center;
+    width: 92vw;
+    height: 100%;
+    padding: 7px;
+    border: 2px solid var(--baby-blue);
+    background-color: var(--white);
+
+    // calc= __images width - gap
+    @include breakpoint(sm) {
+      flex: 1 1 auto;
+      height: auto;
+      align-items: center;
+      align-content: center;
+      flex: 0 1 calc(260px / 2);
+    }
+
+    @include breakpoint(md) {
+      flex: 0 1 calc(350px / 2);
+    }
+
+    @include breakpoint(lg) {
+      flex: 0 1 calc(440px / 3);
+    }
+
+    @include breakpoint(xl) {
+      flex: 0 1 calc(530px / 3);
+    }
+
+    img {
+      width: auto;
+    }
+  }
+
+  &__image:first-child {
+    @include breakpoint(sm) {
+      border-radius: 4px;
+      max-width: 100%;
+      flex: 1 1 100%;
+    }
+  }
+
+  &__name {
+    margin-bottom: 16px;
+  }
+
+  &__description {
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    font-size: 20px;
+    font-family: $font-hn-semibold;
+    color: var(--mid-blue);
+    margin-bottom: 40px;
+  }
+
+  &__colors {
+    margin-bottom: 40px;
+
+    .color-icons {
+      display: flex;
+      flex-flow: row wrap;
+    }
+
+    .color-icon {
+      height: 48px;
+      width: 48px;
+      margin-right: 8px;
+      margin-bottom: 8px;
+      border-radius: 4px;
+
```

Comment:
Please add a `box-shadow` to the color container so that it's easier see the light colors. Something like this:
```
box-shadow: inset 0px 0px 0px 1px rgba(199, 209, 252, 0.25);
```